extension/src/externalresources/ExternalResources.ts

```diff
@@ -110,9 +117,19 @@ export class BlobContainer implements BlobContainerInterface {
           new Error(`${errorMessage} Error: ${err.message}`)
         );
```

Comment:
Why not handle this error in the same way as below? (push to failed and continue...)

Reply:
Good suggestion I'll test it out!